packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -0,0 +1,146 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:vector_math/vector_math_64.dart';
+
+import 'object.dart';
+import 'sliver.dart';
+
+/// A sliver that places multiple sliver children in a linear array along the cross
+/// axis.
+///
+/// Since the extent of the viewport in the cross axis direction is finite,
+/// each sliver is forced to divide up the extent per specification via
+/// [ParentDataWidget] [SliverConstrainedCrossAxis] and [SliverCrossAxisExpanded].
```

Comment:
I am not sure I understand. What does this mean? Are other slivers not allowed to be used in this sliver?

Reply:
This was an outdated comment, yep other slivers are allowed.